bindings/rust/s2n-tls-tokio/tests/common/mod.rs

```diff
@@ -61,14 +63,24 @@ pub fn server_config() -> Result<config::Builder, Error> {
 
 pub fn client_config_tls12() -> Result<config::Builder, Error> {
     let mut builder = config::Config::builder();
-    builder.set_security_policy(&DEFAULT)?;
+    if matches!(fips_mode().unwrap(), FipsMode::Enabled) {
+        builder.set_security_policy(&Policy::from_version("20240502").unwrap())?;
+    } else {
+        builder.set_security_policy(&Policy::from_version("20240501").unwrap())?;
+    }
```

Comment:
I don't think the actual policy really matters here as long as it negotiates TLS 1.2, so I'd suggest just picking one.
```suggestion
    builder.set_security_policy(&Policy::from_version("20240501").unwrap())?;
```

I also think the previous behavior was manually setting "default" anyway, so I don't think it was swapping to FIPS:
https://github.com/aws/s2n-tls/blob/18857b104f003cb2569182b0da0fb2224bcdb10d/bindings/rust/s2n-tls/src/security.rs#L89

Reply:
So I originally though of that but I dont think we can simplify because:

- We currently run Bindings test for Fips and non-Fips and we will use a FIPS policy when linked to FIPS libcrypto
- This is a general helper fn that is used by many other test [client_config](https://github.com/search?q=repo%3Aaws%2Fs2n-tls%20client_config_tls12&type=code), [server_config](https://github.com/search?q=repo%3Aaws%2Fs2n-tls+server_config_tls12&type=code), some of which could be testing FIPS.